src/main/webapp/app/exercises/programming/shared/instructions-render/extensions/programming-exercise-task.extension.ts

```diff
@@ -132,7 +128,9 @@ export class ProgrammingExerciseTaskExtensionWrapper implements ArtemisShowdownE
                 return testsForTask.reduce(
                     (acc: string, { completeString: task, id }): string =>
                         // Insert anchor divs into the text so that injectable elements can be inserted into them.
-                        acc.replace(new RegExp(escapeStringForUseInRegex(task), 'g'), taskContainer.replace(idPlaceholder, id.toString())),
+                        // Without class="d-flex" the injected components height would be 0.
+                        // Added zero-width space as content so the div actually consumes a line to prevent a <ol> display bug in Safari
+                        acc.replace(new RegExp(escapeStringForUseInRegex(task), 'g'), `<div class="pe-task-${id.toString()} d-flex">&#8203;</div>`),
```

Comment:
```suggestion
                        acc.replace(new RegExp(escapeStringForUseInRegex(task), 'g'), `<div class="pe-task-${id.toString()} d-flex">&nbsp;</div>`),
```
This replacement should be possible and is at least in my opinion way more readable

Reply:
I don't think `&nbsp` is the same as `&#8203`
see e.g. https://stackoverflow.com/questions/2046530/whats-the-opposite-of-a-nbsp